graphql/src/application/use-cases/articles/FetchArticlesUseCase.ts

```diff
@@ -0,0 +1,6 @@
+import type { Article } from "../../../domain/articles/Article";
+import * as articleRepository from "../../../infrastructure/repositories/ArticleRepositoryImpl";
+
+export const execute = async (page: number): Promise<Article[]> => {
+  return articleRepository.fetchArticles(page);
+};
```

Comment:
usecaseのメソッドは以下のように変更して

```suggestion
export const fetchArticlesUseCase = async (page: number): Promise<Article[]> => {
  return articleRepository.fetchArticles(page);
};
```

Reply:
use-caseのメソッドを提案されたフォーマットに修正しました。

https://github.com/ksugawara61/expo-template/commit/63152ad7f21a541419f484affe40678dcb29bc02

🤖 Generated by MCP